ui/src/app/cohort-search/cohort-search/cohort-search.component.ts

```diff
@@ -38,6 +38,7 @@ export class CohortSearchComponent implements OnInit, OnDestroy {
   @select(chartData) chartData$: Observable<List<any>>;
   @select(isRequstingTotal) isRequesting$: Observable<boolean>;
   @select(activeCriteriaType) criteriaType$: Observable<string>;
+  @select(s => s.get('initShowChart')) initShowChart$: Observable<boolean>;
```

Comment:
immutable.js methods that involve getting / reading values usually take a `default` parameter that is returned if the attribute can't be found; here we may want to consider `s.get('initShowChart', true)`.

On the one hand, if the state is corrupted somehow then this section of the app won't crash.  On the other hand, a default value here could hide corruption by hiding the crash.